src/collection/views/EditCollection.tsx

```diff
@@ -152,8 +153,23 @@ const EditCollection: FunctionComponent<EditCollectionProps> = props => {
 		});
 	};
 
+	function getValidationErrorForCollection(collection: Avo.Collection.Response): string {
+		if ((collection.description || '').length > MAX_SEARCH_DESCRIPTION_LENGTH) {
+			return `De korte beschrijving is te lang (max ${MAX_SEARCH_DESCRIPTION_LENGTH} karakters)`;
+		}
+		return '';
+	}
+
 	const renderEditCollection = (collection: Avo.Collection.Response) => {
 		async function onSaveCollection() {
+			// Validate collection before save
+			const validationError = getValidationErrorForCollection(currentCollection);
+			if (validationError) {
+				// TODO show error toast
```

Comment:
Toast PR is gemerged dus als je de 0.5.0 branch nog eens binnentrekt kan je die gebruiken.